src/Mvc/Mvc.Core/src/ModelBinding/Binders/CollectionModelBinder.cs

```diff
@@ -127,10 +127,10 @@ public virtual async Task BindModelAsync(ModelBindingContext bindingContext)
             Logger.FoundNoValueInRequest(bindingContext);
 
             // If we failed to find data for a top-level model, then generate a
-            // default 'empty' model (or use existing Model) and return it.
+            // default 'empty' model (or use existing Model when not null or a default value is available) and return it.
             if (bindingContext.IsTopLevelObject)
             {
-                if (model == null)
+                if (model == null && !bindingContext.ModelMetadata.HasDefaultValue)
```

Comment:
Can you point me to where the default value is consumed?

Reply:
https://github.com/dotnet/aspnetcore/blob/7f40fef6d6590d4b7ae6e6dd443c069af6e12f46/src/Mvc/Mvc.Core/src/Infrastructure/ControllerActionInvoker.cs#L551